[2024-05-21T14:41:21.672+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:41:21.697+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-04-30T00:00:00+00:00 [queued]>
[2024-05-21T14:41:21.706+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-04-30T00:00:00+00:00 [queued]>
[2024-05-21T14:41:21.707+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:41:21.725+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2023-04-30 00:00:00+00:00
[2024-05-21T14:41:21.731+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9115) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:41:21.733+0000] {standard_task_runner.py:63} INFO - Started process 9122 to run task
[2024-05-21T14:41:21.733+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'generate_random_numbers', 'scheduled__2023-04-30T00:00:00+00:00', '--job-id', '3210', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpi8s6kdpf']
[2024-05-21T14:41:21.734+0000] {standard_task_runner.py:91} INFO - Job 3210: Subtask generate_random_numbers
[2024-05-21T14:41:21.783+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-04-30T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:41:21.914+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2023-04-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-30T00:00:00+00:00'
[2024-05-21T14:41:21.916+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:41:21.945+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:41:21.947+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:41:21.956+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=generate_random_numbers, run_id=scheduled__2023-04-30T00:00:00+00:00, execution_date=20230430T000000, start_date=20240521T144121, end_date=20240521T144121
[2024-05-21T14:41:21.987+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:41:22.012+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:41:22.018+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
